keyboards/elmospace/m0110a_usb/config.h

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 elmo-space<mail@elmo.space>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xA68C
+#define PRODUCT_ID 0xDB4B
+#define DEVICE_VER 0x0001
+#define MANUFACTURER APPLE
```

Comment:
Not sure about this one...

Reply:
I just checked and the M0110a serial to usb converter also has Apple for the manufacturer in the config.h file so idk

https://github.com/qmk/qmk_firmware/blob/master/keyboards/converter/m0110_usb/config.h

Reply:
Then that should really be changed too, I guess. But not in this PR obviously.